### DISCUSSION TITLE
Education benefits

### DISCUSSION THREAD

**Discussion by sanjitchodankar** [community/community#130755] [GitHub Education]

**Comment by Jakk626** [comment]
I'm looking for the same thing myself but the teachers don't want to teach the learners who want to learn.
Looks like I'll be looking around the darkweb, much more help on there. Tools and instructions also. Didn't want to go that route but big bad coders on Git are useless, half don't even know what they're talking about themselves hence so many questions about "how to?"

Wasting your time on Git, wait until you find a good guy who shares his work, it's usually the best work anyway.

Sent from Proton Mail Android

-------- Original Message --------
